programmers/2일차/수박수박수박수.js

```diff
@@ -0,0 +1,23 @@
+function solution (n) {
+    const word  = "수박";
+    const split_word = word.split("");
+    const answer =[];
+    let count = 0;
+    while(n>count) {
+        for (let i of split_word) {
+            answer.push(i);
+            count += 1;
+            if ( count>= n) {
+                break;
+            }
+        } 
+    }
+    const join_answer = answer.join("");
+    return join_answer ;
```

Comment:
```suggestion
function solution(n) {
    let answer = ""
        
    for (let i = 0; i < n; i++) {
            if (i % 2 ) answer += '박';
            else answer += '수';
        }
    
    return answer;
}
```
이건 제 풀이인데  한번 참고해보시는 것도 좋을 것 같습니다 ㅎㅎㅎ